nodejs12.x-image/cookiecutter-aws-sam-hello-nodejs-lambda-image/{{cookiecutter.project_name}}/hello-world/Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM public.ecr.aws/lambda/nodejs:12
 
-COPY app.js package.json ./
+COPY app.js package*.json ./
 
 RUN npm install
+# If you are building your code for production, instead include a package-lock.json file on this directory and use:
+# RUN npm ci --production
```

Comment:
thoughts on whether this message is enough or whether we should also add it to this section of the README's for visibility? https://github.com/aws/aws-sam-cli-app-templates/tree/master/nodejs12.x-image/cookiecutter-aws-sam-hello-nodejs-lambda-image/%7B%7Bcookiecutter.project_name%7D%7D#use-the-sam-cli-to-build-and-test-locally

Reply:
I am ok with either. Might be nice to have this in multiple places.